packages/cli/src/handlers/dbt/compile.ts

```diff
@@ -0,0 +1,28 @@
+import { ParseError } from '@lightdash/common';
+import execa from 'execa';
+
+export type DbtCompileOptions = {
+    profilesDir: string;
+    projectDir: string;
+    target: string | undefined;
+    profile: string | undefined;
+    select: string[] | undefined;
+    models: string[] | undefined;
+    vars: string | undefined;
+    threads: string | undefined;
+    noVersionCheck: boolean | undefined;
+    exclude: string | undefined;
+    selector: string | undefined;
+    state: string | undefined;
+    fullRefresh: boolean | undefined;
+};
+export const dbtCompile = async (
+    options: DbtCompileOptions,
```

Comment:
the options aren't used. aren't `args` and `options` the same?

Reply:
Yes. Was trying to keep the code similar to the dbt run command. 
If we use the options object we need to convert the properties from camelcase to kebabcase and prefix `--` which is doable.

Reply:
Ah yes I see, whatever you think is simpler!